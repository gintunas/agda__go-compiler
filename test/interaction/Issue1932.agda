-- Andreas, 2016-04-13 display forms do not work for projection patterns

-- {-# OPTIONS -v tc.display:100 #-}

postulate B : Set

module M (A : Set) where
  record R : Set where
    field f : A

open M B

test : R
test = {!!}  -- C-c C-c RET  split on result here

-- Is:
-- M.R.f test = ?
--
-- Expected:
-- R.f test = ?
