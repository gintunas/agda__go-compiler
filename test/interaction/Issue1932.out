(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : R " nil)
((last . 1) . (agda2-goals-action '(0)))
((last . 2) . (agda2-make-case-action '("M.R.f test = ?")))
((last . 1) . (agda2-goals-action '(0)))
